packages/admin/cms-admin/src/pages/createEditPageNode.tsx

```diff
@@ -316,10 +318,55 @@ export function createEditPageNode({
                                                 return <Typography>/</Typography>;
                                             }
 
+                                            function createCompletePath(slug: string) {
+                                                return (
+                                                    <>
+                                                        {parentPath ?? ""}
+                                                        <strong>/{slug}</strong>
+                                                    </>
+                                                );
+                                            }
+
                                             return (
-                                                <Typography>
-                                                    {parentPath}/{values.slug}
-                                                </Typography>
+                                                <>
+                                                    <Typography>{createCompletePath(values.slug)}</Typography>
+                                                    {mode === "edit" && dirtyFields.slug && (
+                                                        <Box mt={2}>
+                                                            <Field name="createRedirectFromOldToNewSlug" type="checkbox">
+                                                                {(props) => (
+                                                                    <FormControlLabel
+                                                                        label={
+                                                                            <Typography display="flex" alignItems="center">
+                                                                                <FormattedMessage
+                                                                                    tagName="span"
+                                                                                    id="comet.pages.pages.page.createRedirectFromOldToNewSlug"
+                                                                                    defaultMessage="Create redirect from {oldSlug} to {newSlug}"
+                                                                                    values={{
+                                                                                        oldSlug: createCompletePath(initialValues?.slug ?? ""),
+                                                                                        newSlug: createCompletePath(values.slug),
+                                                                                    }}
```

Comment:
```suggestion
                                                                                    defaultMessage="Create redirect from {oldPath} to {newPath}"
                                                                                    values={{
                                                                                        oldPath: createCompletePath(initialValues?.slug ?? ""),
                                                                                        newPath: createCompletePath(values.slug),
                                                                                    }}
```

Reply:
I removed the paths from the label altogether because they clutter the label and it gets very long